test/test_mt940.rb

```diff
@@ -9,6 +17,8 @@ class TestMt940 < Test::Unit::TestCase
   def test_it_should_parse_fixture_files_correctly
     Dir[File.dirname(__FILE__) + "/fixtures/*.txt"].reject { |f| f =~ /sepa_snippet/ }.each do |file|
       data = MT940.parse(IO.read(file))
+      data.flatten.each { |field| field.extend(Mt940FieldEncoderWithoutContent) }
```

Comment:
No worries: I just recreated the yaml files and checked the result for plausibility. That's most likely the way this gigantic test was created in the first place. At least this is a smoke test if yaml or ruby are changing and happen to introduce introduce bugs into the parsing.